packages/MsGraph/Actions/FindMeetingTimes.cs

```diff
@@ -66,7 +67,7 @@ protected override async Task<List<CalendarSkillTimeSlotModel>> CallGraphService
             var attendeesProperty = this.AttendeesProperty.GetValue(dcState);
             var duration = this.DurationProperty.GetValue(dcState);
             var timeZoneProperty = this.TimeZoneProperty.GetValue(dcState);
-            var timeZone = TimeZoneInfo.FindSystemTimeZoneById(timeZoneProperty);
+            var timeZone = TZConvert.GetTimeZoneInfo(timeZoneProperty);
```

Comment:
Sorry it may have slipped by me the detail. What specific issue with non-Windows that's caused by TimeZoneInfo? I thought it handles macOS/Linux already.